.github/workflows/maven.yaml

```diff
@@ -110,6 +109,12 @@ jobs:
       run: mvn -B -V -e -ntp "-Dstyle.color=always" -DskipFormat ${{ matrix.profile.args }}
       env:
         MAVEN_OPTS: -Djansi.force=true
+    - name: Build with Maven (ZooKeeper compatibility build)
+      if: ${{ github.base_ref != 'main' }}
+      timeout-minutes: 60
+      run: mvn -B -V -e -ntp "-Dstyle.color=always" -DskipFormat package -DskipTests -Dhadoop.version=3.0.3 -Dzookeeper.version=3.5.10
+      env:
```

Comment:
This just puts inline what we have previously defined elsewhere. It seems the reason to do this is to exclude the main branch, and not use the normal args, but I'm not sure why we're excluding the main branch.